### PR TITLE
[#1130] Add documentation for the exposed Popcorn.options

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/63272/tickets/1130-document-popcornoptions-property#ticket-1130-7

### CODE REVIEW

File: popcorn-constructor/_posts/2012-12-12-popcornoptions.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Popcorn.options
+---
+# Popcorn.options #
+
+## Purpose ##
+
+Provides access to all of the options that were passed in with the optionsObject upon creation of the Popcorn instance.
```
Comment (dseif) [inline popcorn-constructor/_posts/2012-12-12-popcornoptions.md]: optionsObject should link to what it's talking about

File: popcorn-constructor/_posts/2012-12-12-popcornoptions.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+## Example ##
+
+[Live video showing how to console.log the options values ](http://jsfiddle.net/popcornjs/uSpJH/)
```
Comment (dseif) [inline popcorn-constructor/_posts/2012-12-12-popcornoptions.md]: What about defaults?

File: popcorn-constructor/_posts/2012-12-12-popcornoptions.md (hunk 1 above)
Comment (rwaldron) [inline popcorn-constructor/_posts/2012-12-12-popcornoptions.md]: +1, a link to this: http://popcornjs.org/popcorn-docs/popcorn-constructor/

File: popcorn-constructor/_posts/2012-12-12-popcornoptions.md (hunk 2 above)
Comment (rwaldron) [inline popcorn-constructor/_posts/2012-12-12-popcornoptions.md]: I'm not sure this makes sense... `Popcorn.options` looks like a "static" property on the `Popcorn` constructor object

Comment (dseif) [inline popcorn-constructor/_posts/2012-12-12-popcornoptions.md]: yea I forgot you can just do `instance.defults( "pluginName", { /* defaults */ });` looks good to me!

**Pull request merged**